### PR TITLE
ref: addition of string-input

### PR DESCRIPTION
**Pull request by drjova**
* Ref inspirehep/inspirehep#1255

### CODE REVIEW

File: src/ref-field/ref-field.component.html [hunk 1]
```diff
@@ -8,7 +8,8 @@
       </div>
     </div>
     <div *ngIf="!isTemplateEnabled">
-      <a target="_blank" class="break-word" [href]="anchorHref">{{anchorDisplay}}</a>
+      <string-input *ngIf="shouldDisplayInputField" [pathString]="pathString" [value]="ref" [tabIndex]="tabIndex" [placeholder]="schema.title"></string-input>
```
Comment (harunurhan) [inline src/ref-field/ref-field.component.html:11]: I think this shouldn't be under `*ngIf="!isTemplateEnabled"`, we might want to display template (currently not used) even if `displayInputField` is enabled.

Comment (harunurhan) [reply src/ref-field/ref-field.component.html:11]: `pathString` for this should be `<pathString>/$ref`

Comment (harunurhan) [reply src/ref-field/ref-field.component.html:11]: Depending on the schema and what makes sense, maybe it's better to use `schema.properties.$ref.title` instead as placeholder.

Comment (harunurhan) [reply src/ref-field/ref-field.component.html:11]: `[value]` should be actual `$ref` value, not hardcoded string.

**Review by harunurhan — CHANGES_REQUESTED** [review]

File: src/ref-field/ref-field.component.ts [hunk 2]
```diff
@@ -100,6 +100,10 @@ export class RefFieldComponent implements OnChanges {
     return this.appGlobalsService.templates[this.refConfig.templateName];
   }
 
+  get pathStringWithRef(): string {
+    return this.pathString + '/$ref';
```
Comment (harunurhan) [inline src/ref-field/ref-field.component.ts:104]: You should use `PathUtilService` for this.

Comment (drjova) [reply src/ref-field/ref-field.component.ts:104]: interesting 😄

**Review by harunurhan — APPROVED** [review]

**Pull request merged**